packages/aws-rfdk/lib/core/lib/mountable-efs.ts

```diff
@@ -66,6 +66,18 @@ export interface MountableEfsProps {
    * @default No extra options.
    */
   readonly extraMountOptions?: string[];
+
+  /**
```

Comment:
There's a downside here that we should mention... If you have mount targets in multiple AZs, then the Route53 resolver will always give you the mount target IP that's in the same AZ as the requester (or random if the request isn't coming from an IP that shares an AZ with a resolution).

When resolving via IP address like this you don't get that AZ-matching. Thus, cross-AZ data charges may be incurred.

Reply:
This code actually does performs AZ matching. See:

https://github.com/jusiskin/aws-rfdk/blob/daa2fd714859be3cf31c45a394ec883f8dfa7f94/packages/aws-rfdk/lib/core/scripts/bash/mountEfs.sh#L78

and

https://github.com/jusiskin/aws-rfdk/blob/daa2fd714859be3cf31c45a394ec883f8dfa7f94/packages/aws-rfdk/lib/core/scripts/bash/mountEfs.sh#L85

Reply:
Yeah. I saw that, but didn't come back to remove the comment. My bad.